lua/nordbuddy.lua

```diff
@@ -1,341 +1,498 @@
+        'Delimiter', -- VL
+        'vimParenSep', 'vimSep', -- vim
+        'shCmdParenRegion', 'shCmdSubRegion' -- sh
+    }
+    local repeats = {
+        'TSRepeat', -- TS
+        'Repeat' -- VL
+    }
+    local strings = {
+        'TSCharacter', 'TSString', -- TS
+        'Character', 'String', -- VL
+        'cString', 'cCppString', -- C/C++
+        'pythonstringdelimiter', 'pythonbytes', 'pythonrawbytes',
+        'pythonbytescontent' -- python
+    }
+    local strings_specials = {
+        'TSStringRegex', 'TSStriingEscape', -- TS
+        'SpecialChar', -- VL
+        'cSpecialCharacter', -- C/C++
+        'pythonescape' -- python
+    }
+    local symbols = {
+        'TSSymbol', -- TS
+        'Special', 'SpecialComment', -- VL
+        'cSpecial', -- C/C++
+        'pythonsymbol', -- python
+        'rustCommentLineDoc', 'rustCommentBlockDoc' -- rust
+    }
+    local texts = {
+        'TSText', 'TSStrong', 'TSEmphais', 'TSUnderline', 'TSStrike', 'TSTitle',
+        'TSLiteral', -- TS
+        'Text', -- VL
+        'manTitle' -- man
+    }
+    local uris = {
+        'TSURI' -- TS
+    }
+    local types = {
+        'TSType', 'TSTypeBuildin', -- TS
+        'Type', 'StorageClass', 'Structure', 'Typedef', -- VL
+        'cType', 'cStorageClass', 'cStructure', 'cppType', 'cppStorageClass',
+        'cppStructure', -- C/C++
+        'pythonclass', -- python
+        'vimVar' -- vim
+    }
+    local variables = {
+        'TSVariable', -- TS
+        'Identifier', -- VL
+        'pythonselfarg', -- python
+        'rustIdentifier', -- rust
+        'shDerefSimple', 'shDerefVar' -- sh
+    }
+    local builtins = { -- such as self. in python
+        'TSConstBuiltin', 'TSFuncBuiltin', 'TSVariableBuiltin', -- TS
+        'pythonbuiltin', 'pythonself' -- python
+    }
+    local notes = {
+        'TSNote', -- TS
+        'Todo', -- VL
+        'cTODO' -- C/C++
+    }
+    local warnings = {
+        'TSWarning', -- TS
+        'Debug' -- VL
+    }
+    local dangers = {
+        'TSDanger' -- TS
+    }
+    local errors = {
+        'TSError', -- TS
+        'Error', -- VL
+        'rustCommentLineDocError', 'rustCommentBlockDocError' -- rust
+    }
+    local groups = {
+        {attributes, c.nord9}, --
+        {numbers, c.nord15}, --
+        {comments, c.nord3:light()}, --
+        {constructors, c.nord3:light()}, --
+        {conditionals, c.nord10, cno, i}, --
+        {constants, c.nord4}, --
+        {defines, c.nord10}, --
+        {exceptions, c.nord9}, --
+        {fields, c.nord9, cno, i}, --
+        {functions, c.nord4, cno, i}, --
+        {includes, c.nord10}, --
+        {keywords, c.nord9, cno, sno}, --
+        {labels, c.nord10, cno, i}, --
+        {namespaces, c.nord4, cno, i}, --
+        {nones, c.nord4}, --
+        {operators, c.nord9}, --
+        {parameters, c.nord4, cno, i}, --
+        {properties, c.nord10}, --
+        {punctuations, c.nord9}, --
+        {repeats, c.nord10, cno, i}, --
```

Comment:
This one is up to you, not a big deal for me either way.

```suggestion
        {repeats, c.nord8, cno, i}, --
```